build.mill

```diff
@@ -95,12 +95,7 @@ trait AWSSpec extends Cross.Module[String] with BaseModule {
   override def artifactName: T[String] = T(s"aws-$service-spec")
 
   override def ivyDeps: T[Agg[Dep]] = Agg(
-    ivy"software.amazon.smithy:smithy-aws-traits:$smithyVersion",
-    ivy"software.amazon.smithy:smithy-aws-cloudformation-traits:$smithyVersion",
-    ivy"software.amazon.smithy:smithy-aws-iam-traits:$smithyVersion",
-    ivy"software.amazon.smithy:smithy-waiters:$smithyVersion",
-    ivy"software.amazon.smithy:smithy-aws-endpoints:$smithyVersion",
-    ivy"software.amazon.smithy:smithy-aws-smoke-test-model:$smithyVersion"
+    ivy"software.amazon.smithy:smithy-aws-traits:$smithyVersion"
```

Comment:
turns out we don't need any of the others anymore, if we remove the namespaces (as seen below). Which seems to make sense to me